dir.props

```diff
@@ -60,6 +62,41 @@
     <NuGetIngoreLockFile>true</NuGetIngoreLockFile>
   </PropertyGroup>
 
+  <!--
+    Set up Roslyn predefines
+  -->
+  <PropertyGroup>
+    <RoslynPackageDir>$(PackagesDir)/$(RoslynPackageName).$(RoslynVersion)/</RoslynPackageDir>
+    <RoslynPropsFile>$(RoslynPackageDir)build/Microsoft.Net.ToolsetCompilers.props</RoslynPropsFile>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(OsEnvironment)'=='Unix' and '$(UseRoslynCompiler)'=='true'">
+    <!--
+      PDB support isn't implemented yet.
+      Note that both DebugSymbols and DebugType need set or project references will assume they need to copy pdbs and fail.
+    -->
+    <DebugSymbols>false</DebugSymbols>
+    <DebugType>none</DebugType>
+    <!-- Delay signing with the ECMA key currently doesn't work. -->
+    <UseECMAKey>false</UseECMAKey>
```

Comment:
Delay signing with normal keys doesn't fail. In fact, it is necessary to get the right full assembly names. It falls apart hard in tests with InternalsVisibleTo if it's off. Also, open source signing is lumped in with this.